ts/DataGrid/DataGrid.ts

```diff
@@ -397,7 +400,25 @@ class DataGrid {
     // ---------------- Private methods
 
     /**
-     * Determs whether a column is editable or not.
+     * Check which columns should be displayed based on the individual
+     * column `show` option.
+     * @internal
+     */
+    private getColumnsToDisplay(): Array<string> {
+        const columnsOptions = this.options.columns,
+            tableColumns = this.dataTable.getColumnNames();
+
+        return tableColumns.filter(function (columnName): boolean {
+            const column = columnsOptions[columnName];
+            if (column && defined(column.show)) {
```

Comment:
Do we need defined here?

Reply:
Yes, we need it ;) 
False is a valid value here, thus we have to use `define`.